include/ranges.h

```diff
@@ -117,13 +117,13 @@ class coordinate {
 #undef CELERITY_DETAIL_DEFINE_COORDINATE_BINARY_COPY_LHS_OPERATOR
 
 #define CELERITY_DETAIL_DEFINE_COORDINATE_BINARY_INPLACE_LHS_OPERATOR(op)                                                                                      \
-	friend constexpr Interface& operator op(Interface& lhs, const Interface& rhs) {                                                                            \
+	friend constexpr Interface& operator op(Interface & lhs, const Interface & rhs) {                                                                          \
```

Comment:
Any idea what's happening here?  
`Interface &` is weird in the first place, but it's very weird to have both that and `Interface&` in the same line?

Seems to happen only in the Macro definition. We could of course just ignore this.

Reply:
I believe that clang is unable to parse this properly because the context is not clear from inside the macro definition.